xmtp_mls/src/groups/sync.rs

```diff
@@ -345,7 +343,7 @@ impl MlsGroup {
                 );
                 if let Err(err) = openmls_group.merge_staged_commit(&provider, pending_commit) {
                     log::error!("error merging commit: {}", err);
-                    conn.set_group_intent_to_publish(intent.id)?;
```

Comment:
Interesting, I wonder if this was a bug before? In the previous code it would have continued to L371 and set it to committed